docs/notebooks/asynchronous_batch_bo.pct.py

```diff
@@ -0,0 +1,197 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.12.0
+#   kernelspec:
+#     display_name: 'Python 3.7.5 64-bit (''.venv'': venv)'
+#     name: python3
+# ---
+
+# %% [markdown]
+# # Asynchronous batch Bayesian optimization
+#
+# As shown in [Asynchronous Bayesian Optimization](asynchronous_bayesian_optimization.ipynb) tutorial, Trieste provides support for running observations asynchronously. In that tutorial we used greedy batch acquisition function called Local Penalization, and collected one new point whenever an observation was received. We also used Python multiprocessing module to run distributed observations in parallel.
```

Comment:
In that tutorial we used *a* greedy batch acquisition function called Local Penalization, and *requested* one new point whenever an observation was received. We also used *the* Python multiprocessing module to run distributed observations in parallel.